ui-v2/app/utils/form/builder.js

```diff
@@ -0,0 +1,156 @@
+import { get, set, computed } from '@ember/object';
+import Changeset from 'ember-changeset';
+import lookupValidator from 'ember-changeset-validations';
+
+// Keep these here for now so forms are easy to make
+// TODO: Probably move this to utils/form/parse-element-name
+import parseElementName from 'consul-ui/utils/get-form-name-property';
+const defaultChangeset = function(data, validators) {
+  const changeset = new Changeset(data, lookupValidator(validators), validators);
+  // TODO: Currently supporting ember-data nicely like this
+  changeset.isSaving = computed('data.isSaving', function() {
+    return get(this, 'data.isSaving');
+  });
+  return changeset;
+};
+/**
+ * Form builder/Form factory (WIP)
```

Comment:
Seeing WIP on the master branch is never reassuring.

Reply:
No prob, removed in https://github.com/hashicorp/consul/pull/5021